runtime/doc/treesitter.txt

```diff
@@ -1056,7 +1051,9 @@ contents.
 To create a LanguageTree (parser object) for a given buffer and language, use:
 
 >lua
-    local parser = vim.treesitter.get_parser(bufnr, lang)
+
+     local parser = vim.treesitter.get_parser(bufnr, lang)
+
```

Comment:
so much whitespace 😂

Reply:
The doc workflow is a bit of a rube goldberg machine. Changing one thing can have unexpected results somewhere else.